opflexagent/as_metadata_manager.py

```diff
@@ -560,6 +560,7 @@ def conn_track_del(self, netns):
     def conn_track_config(self, netns):
         snatstr = "\n".join([
             "[program:opflex-conn-track-%s]" % netns,
+            "process_name=opflex-conn-track-%s" % netns,
```

Comment:
do we have to worry about this guy also then since I don't see process_name attribute there either?
https://github.com/noironetworks/python-opflex-agent/blob/master/opflexagent/as_metadata_manager.py#L510

Reply:
The diagnosis was incorrect - pushed an update with the correct fix.